### ISSUE TITLE
Initial breakaway alert is not uttered when temperature reaches "very hot".

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#85]
After initial testing of #81 by listening, I examined the alerts in the Activity Log in the A11y View and see that the initial break away alert is not actually uttered, only the second alert that contains the word "more" is showing up in the Activity Log. I never see the initial alert, "Very hot. Atoms break away from chemistry book", and only see, "Very Hot. More atoms break away from chemistry book."

I do think it will be useful to have logic for at least one follow-up breakaway alert, though I think the alerts are going to get shorter.

**Comment by terracoda** [comment]
![breakaway-missing](https://user-images.githubusercontent.com/1902116/43730572-532eea84-9986-11e8-907f-44fcbe6117b1.png)

**Comment by terracoda** [comment]
In the above screen shot, I think the 4th bullet should be: " Very hot. Atoms break away from chemistry book" rather than " Atoms jiggling very fast, temperature very hot."

**Comment by jessegreenberg** [comment]
Should be fixed in the above commit. Closing.
